README.md

```diff
@@ -19,4 +19,9 @@ In the meantime, you can install and use this the Legacy OmniSharp extension to
     * Click the [X] to the right of the C# extension.
     * Allow Visual Studio Code to restart. 
     
-3. This extension has a dependency on Mono >= 4.0.1.
\ No newline at end of file
+3. This extension has a dependency on Mono >= 4.0.1.
+
+4. This extension can be used to target ASP.NET 5 RC1 and older DNX-based projects. If you need to install ASP.NET 5 RC1, you can still find it in the following locations:
```

Comment:
There *is* an error from the extension:

```
[ERROR:OmniSharp.Dnx.DnxPaths] The specified runtime path 'default' does not exist. Searched locations C:\Users\DustinCa\.dnx\runtimes\dnx-clr-win-x86.default
C:\Users\DustinCa\.dnx\runtimes\kre-clr-win-x86.default
C:\Users\DustinCa\.dnx\packages\KRE-CLR-x86.default
C:\Users\DustinCa\.dnx\runtimes\dnx-clr-win-x86.default
C:\Users\DustinCa\.dnx\runtimes\kre-clr-win-x86.default
C:\Users\DustinCa\.dnx\packages\KRE-CLR-x86.default
C:\Users\DustinCa\.k\runtimes\dnx-clr-win-x86.default
C:\Users\DustinCa\.k\runtimes\kre-clr-win-x86.default
C:\Users\DustinCa\.k\packages\KRE-CLR-x86.default
C:\Users\DustinCa\.kre\runtimes\dnx-clr-win-x86.default
C:\Users\DustinCa\.kre\runtimes\kre-clr-win-x86.default
C:\Users\DustinCa\.kre\packages\KRE-CLR-x86.default.
Visit https://github.com/aspnet/Home for an installation guide.
```

However,none of those folders exists for me and it carries on just fine.

```
[INFORMATION:OmniSharp.Dnx.DnxProjectSystem] Scanning 'c:\Projects\CSharpEssentials' for DNX projects
[INFORMATION:OmniSharp.Dnx.DnxProjectSystem] No project.json based projects found
[INFORMATION:OmniSharp.MSBuild.MSBuildProjectSystem] Detecting projects in 'c:\Projects\CSharpEssentials\CSharpEssentials.sln'.
[INFORMATION:OmniSharp.MSBuild.MSBuildProjectSystem] Loading project from 'c:\Projects\CSharpEssentials\Source\CSharpEssentials\CSharpEssentials.csproj'.
[INFORMATION:OmniSharp.MSBuild.MSBuildProjectSystem] Using toolset 2.0 for c:\Projects\CSharpEssentials\Source\CSharpEssentials\CSharpEssentials.csproj
[INFORMATION:OmniSharp.MSBuild.MSBuildProjectSystem] Loading project from 'c:\Projects\CSharpEssentials\Source\CSharpEssentials.Vsix\CSharpEssentials.Vsix.csproj'.
[INFORMATION:OmniSharp.MSBuild.MSBuildProjectSystem] Using toolset 2.0 for c:\Projects\CSharpEssentials\Source\CSharpEssentials.Vsix\CSharpEssentials.Vsix.csproj
```